packages/dds/matrix/src/test/time/sharedMatrix.spec.ts

```diff
@@ -0,0 +1,187 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { benchmark, isInPerformanceTestingMode } from "@fluid-tools/benchmark";
+
+import { SharedMatrix } from "../../index.js";
+import { createLocalMatrix } from "../utils.js";
+
+describe("SharedMatrix execution time", () => {
+	// The value to be set in the cells of the matrix.
+	const matrixValue = "cellValue";
+	// The test matrix's size will be 0*0, 10*10, 100*100, 1000*1000.
+	const matrixSizes = isInPerformanceTestingMode
+		? [10, 100, 1000]
+		: // When not measuring perf, use a single smaller data size so the tests run faster.
+			[10];
+
+	// The number of operations to perform on the matrix.
+	const operationCounts = isInPerformanceTestingMode
+		? [10, 100, 1000]
+		: // When not measuring perf, use a single smaller data size so the tests run faster.
+			[5];
+
+	let localMatrix: SharedMatrix | undefined;
+
+	// IMPORTANT: variables scoped to the test suite are a big problem for memory-profiling tests
+	// because they won't be out of scope when we garbage-collect between runs of the same test,
+	// and that will skew measurements. Tests should allocate all the memory they need using local
+	// variables scoped to the test function itself, so several iterations of a given test can
+	// measure from the same baseline (as much as possible).
+
+	beforeEach(async () => {
+		// CAREFUL: usually beforeEach/afterEach hooks are used to initialize or interact with variables
+		// whose scope is the encompasing test suite, but that's a problem for memory-profiling tests.
+		// See the comment at the top of the test suite for more details.
+	});
+
+	afterEach(() => {
+		// CAREFUL: usually beforeEach/afterEach hooks are used to initialize or interact with variables
+		// whose scope is the encompasing test suite, but that's a problem for memory-profiling tests.
+		// See the comment at the top of the test suite for more details.
+	});
```

Comment:
In this case we should remove all this, since it applies to memory tests, but not execution time ones.

Reply:
Removed